<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundDialog</class>
 <widget class="QDialog" name="BackgroundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tło</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonBkg0">
     <property name="text">
      <string>Jednolite białe tło</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonBkg1">
     <property name="text">
      <string>Jednolite ciemne tło</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonBkg2">
     <property name="text">
      <string>Niejednolite tło (brak drewna)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonBkg3">
     <property name="text">
      <string>Niejednlite tło (elementy drewna)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonBkg4">
     <property name="text">
      <string>Jednolite tło koloru drewna</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxBkg">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
